### PR TITLE
Update repo_name Enum for new 'beta' repo

### PR DESCRIPTION
**Pull request by JayH5**
#### Pull Request (PR) description
Along with the APT repo update (#150), the repo name seems to have changed from `testing` to `beta` for beta releases.

See http://docs.grafana.org/installation/debian/#apt-repository

#### This Pull Request (PR) fixes the following issues

I haven't created an issue.

### CODE REVIEW

**Comment by bastelfreak** [comment]
Hey @JayH5, could you add an acceptance test for this?

**Comment by bastelfreak** [comment]
Can you take a look at the failing travis jobs?

File: manifests/install.pp [hunk 1]
```diff
@@ -98,9 +98,15 @@
           }
 
           if ( $::grafana::manage_package_repo ){
+            # http://docs.grafana.org/installation/rpm/#install-via-yum-repository
+            $baseurl = $::grafana::repo_name ? {
```
Comment (bastelfreak) [inline manifests/install.pp]: please don't add leading `::` after the `$`. That is deprecated syntax that was required for puppet 3.

Comment (JayH5) [reply manifests/install.pp]: Yes, I'm aware of that, I was just being consistent with the rest of the file. Will change.

**Comment by JayH5** [comment]
I'm not certain about the RedHat/CentOS stuff as I don't have a lot of experience with the platform. `beta1` is technically the release:
```
[root@351a8e69d3f1 /]# yum info grafana
Loaded plugins: fastestmirror, ovl
Loading mirror speeds from cached hostfile
 * base: centos.mirror.ac.za
 * extras: centos.mirror.ac.za
 * updates: centos.mirror.ac.za
Available Packages
Name        : grafana
Arch        : x86_64
Version     : 6.0.0
Release     : beta1
Size        : 54 M
Repo        : grafana
Summary     : Grafana
URL         : https://grafana.com
License     : "Apache 2.0"
Description : Grafana
```

(Apologies for all the force pushing, keep noticing mess I left behind)

**Comment by bastelfreak** [comment]
Thanks!

**Pull request merged**